style.css

```diff
@@ -135,11 +144,230 @@ body {
   opacity: .5;
 }
 
-.main__image {
-  width: 770px;
-  height: 657px;
-}
-
 .main__title-text_highlight {
   background: #ecf3ff;
 }
+
+                   /* Мобильные устройства */
+
+@media (max-width: 768px) {
+  .menu {
+    display: none;
+ }
+
+  .logo {
+    right: 12px;
+ }
+
+  .logo__circle {
+    transform: translateY(-95%);
+ }
+
+  .logo__circle_color_blue {
+    top: 6px;
+ }
+
+  .logo__text {
+    display: none;
+ }
+
+  .header {
+    margin-top: 51px;
+    margin-bottom: 33px;
+ }
+
+  .menu__item::after {
+    visibility: hidden;
+ } 
```

Comment:
Codacy found an issue: [Unexpected whitespace at end of line (no-eol-whitespace)](https://app.codacy.com/gh/WebHeroSchool/new_project2.0/pullRequest?prid=6297664)